rental.rb

```diff
@@ -0,0 +1,17 @@
+require './person'
+require './book'
+
+class Rental
+  attr_reader :person, :book
+  attr_accessor :date
+
+  def initialize(date)
```

Comment:
- [ ] To be able to set up the has many through association, you still need to provide the `book` and `person` objects.

  ```
  def initialize(date, book, person)
    ...
  ```